pkg/platform/baremetal/baremetal_internal_test.go

```diff
@@ -49,6 +49,12 @@ func TestConfigurationIsInvalidWhen(t *testing.T) {
 		"conntrack_max_per_core_is_negative": func(c *config) {
 			c.ConntrackMaxPerCore = -1
 		},
+		"clc_snippets": func(c *config) {
```

Comment:
I'd break it down into 2 cases, named `clc_snippet_key_is_empty` and `clc_snippet_is_empty`.